enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/CompositeNodeKeyConstraintAcceptanceTest.scala

```diff
@@ -131,25 +139,30 @@ class CompositeNodeKeyConstraintAcceptanceTest extends ExecutionEngineFunSuite w
 
     // Then
     a[CypherExecutionException] should be thrownBy {
-      exec("CREATE CONSTRAINT ON (n:User) ASSERT (n.firstname,n.lastname) IS NODE KEY")
+      testWith(Configs.AbsolutelyAll,
+        "CREATE CONSTRAINT ON (n:User) ASSERT (n.firstname,n.lastname) IS NODE KEY")
     }
   }
 
   test("trying to add duplicate node when node key constraint exists") {
     createLabeledNode(Map("name" -> "A"), "Person")
-    exec("CREATE CONSTRAINT ON (person:Person) ASSERT (person.name) IS NODE KEY")
+    testWith(Configs.Procs,
```

Comment:
I would agree, but the 'given' parts were tested previously as well. Thus I thought there is some reason behind it. But if there is not, I happily change it